components/user_settings/modal/user_settings_modal.tsx

```diff
@@ -292,9 +310,9 @@ class UserSettingsModal extends React.Component {
                                     updateTab={this.updateTab}
                                     closeModal={this.closeModal}
                                     collapseModal={this.collapseModal}
-                                    setEnforceFocus={(enforceFocus) => this.setState({enforceFocus})}
+                                    setEnforceFocus={(enforceFocus: any) => this.setState({enforceFocus})}
                                     setRequireConfirm={
-                                        (requireConfirm, customConfirmAction) => {
+                                        (requireConfirm: any, customConfirmAction: any) => {
```

Comment:
`requireConfirm` is a `boolean`
`customConfirmAction` is a function that takes in another function, should look like: `(() => void) => void)`